UKHomeOffice/cerberus-service
# COP-11003: Updated test case to test Target Information Sheet updates for RoRo freight mode split

## Description
Updated test case to test Target Information Sheet updates for RoRo freight mode split
## To Test
npm run cypress:test:local -- --spec cypress/integration/cerberus/issue-a-target.spec.js

Should submit a target successfully from a RoRo-Accompanied task and it should be moved to target issued tab

Should submit a target successfully from a RoRo-Unaccompanied task and it should be moved to target issued tab

Should submit a target successfully from a RoRo-Tourist task and it should be moved to target issued tab

## Developer Checklist

\* Required
